frontend/mission-components/app/components/ui/HandoverModal.tsx

```
@@ -0,0 +1,157 @@
+                    exit={{ scale: 0.9, opacity: 0, y: 20 }}
+                    className="relative w-full max-w-2xl bg-slate-950 border border-slate-800 shadow-2xl rounded-sm overflow-hidden flex flex-col max-h-[80vh]"
+                >
+                    {/* Header */}
+                    <div className="p-6 border-b border-slate-800 bg-slate-900/50 flex items-center justify-between">
+                        <div className="flex items-center gap-3">
+                            <div className="p-2 rounded bg-cyan-500/10 border border-cyan-500/20 text-cyan-400">
+                                <Clipboard size={20} />
+                            </div>
+                            <div>
+                                <h2 className="text-sm font-bold text-slate-100 uppercase tracking-[0.2em]">Operational Handover</h2>
+                                <p className="text-[10px] text-slate-400 font-mono mt-0.5">TRANSITION_LOG // {handoverData.timestamp}</p>
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)